packages/bot-web-ui/src/components/dashboard/bot-builder/toolbox/toolbox.scss

```diff
@@ -98,22 +108,26 @@
             width: 25px;
         }
     }
+
```

Comment:
@prince-deriv please try formatting the file with option+shift+F in vscode, these extra lines seem to be extra